2_flow_metrics/src/calc_FDC.R

```diff
@@ -21,7 +22,11 @@ calc_FDCmetrics <- function(site_num, clean_daily_flow, yearType,
   #' @param out_format format for the output table of metrics. The default matches
   #' the EflowStats format. Use 'pivot' for a simpler table.
   #' @param threshold_type one of either 'high' or 'low' indicating if the threshold
-  #' should consider all flows higher or lower than the threshold.
+  #' should consider all flows higher or lower than the threshold.false, events are defined such that
+  #' there are no overlapping events between quantile groups.  If false, the same set of dates may
```

Comment:
Check description. Is the `threshold.false` supposed to be for `allow_event_overlap`?